### DISCUSSION TITLE
自动佩戴当前直播间勋章功能，细分出发弹幕自动佩戴选项

### DISCUSSION THREAD

**Discussion by u2shana** [the1812/Bilibili-Evolved#4213] [功能建议 / Ideas]
现在的逻辑是点进去A直播间就戴A牌子
但是有时候会把A丢后台然后点进去B直播间，自动戴了B牌子，这时候在A直播间发弹幕就很容易忘记切回去A牌子

可以的话细分出发弹幕时自动佩戴牌子的功能

但是这样每次发弹幕都会检测切牌子，对性能影响大么？

**Comment by the1812** [comment]
> 但是这样每次发弹幕都会检测切牌子，对性能影响大么？

不合适, 切牌子不一定能比弹幕早切完, 而且 b 站对这个接口限流很严格, 用他官方的勋章面板快速切几下就会被提示操作过快

需要找一个合适的时机检测并再次触发自动佩戴
